src/components/Modals/NewMessageModal.jsx

```diff
@@ -0,0 +1,215 @@
+  };
+
+  /* eslint-disable jsx-a11y/label-has-associated-control */
+  return (
+    <Dialog
+      open={showModal}
+      aria-labelledby="new-message-modal"
+      onClose={() => setShowModal(false)}
+    >
+      <Box
+        noValidate
+        sx={{
+          display: 'flex',
+          flexDirection: 'column',
+          justifyContent: 'center',
+          alignItems: 'center',
+          padding: '20px',
+          minWidth: '50%',
+          minHeight: '90%'
+        }}
+      >
+        <form onSubmit={(e) => handleSubmit(e)} autoComplete="off">
+          <Typography display="flex" justifyContent="center" variant="h5">
+            {oldMessage ? 'Reply To' : 'New Message'}
+          </Typography>
+          <TextField
+            margin="normal"
+            value={message.recipientPodUrl}
+            type="text"
+            name="recipientPodUrl"
+            id="recipientPodUrl"
+            onChange={(e) => handleChange(e)}
+            required
+            autoFocus
+            label="To"
+            fullWidth
+          />
+          <TextField
+            margin="normal"
+            value={message.title}
+            type="text"
+            name="title"
+            id="title"
+            onChange={(e) => handleChange(e)}
+            required
+            label="Subject"
+            inputProps={{ maxLength: '24' }}
+            fullWidth
+          />
+          <TextField
+            margin="normal"
+            value={oldMessage && originalMessage}
+            type="previousMessage"
+            name="previousMessage"
+            id="previousMessage"
+            label="Previous Message"
+            // helperText="Previous message"
+            // variant="filled"
+            multiline
+            rows={3}
+            InputProps={{
+              readOnly: true
+            }}
+            InputLabelProps={{
+              shrink: true
+            }}
+            fullWidth
+          />
+
+          <TextField
+            margin="normal"
+            value={message.message}
+            type="text"
+            name="message"
+            id="message"
+            onChange={(e) => handleChange(e)}
+            multiline
+            rows={6}
+            label="Enter Message"
+            placeholder="Add a description here"
+            required
+            fullWidth
+          />
+          <DialogActions>
+            <Button
+              variant="outlined"
+              color="error"
+              startIcon={<ClearIcon />}
+              onClick={() => setShowModal(false)}
+              fullWidth
+            >
+              CANCEL
+            </Button>
+            <Button
+              variant="contained"
+              type="submit"
+              color="primary"
+              startIcon={<CheckIcon />}
+              fullWidth
+            >
+              Submit
+            </Button>
+          </DialogActions>
+          {error && <div>{error}</div>}
```

Comment:
I would say maybe make the error message red, and the success message green so they stand out more. But, I think Russell is working on the status notification messaging --> snackbar feature.

So, maybe make sure this modal uses the same notification components as the others - that way it'll work when that is updated/merged??

Reply:
Yeah my idea was to wait until the status notification is reworked more, but I can definitely take another look at that.